concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/DelegationStakeDecreased.java

```diff
@@ -22,6 +22,6 @@ public class DelegationStakeDecreased extends AbstractDelegatorResult {
 
     @Override
     public TransactionResultEventType getType() {
-        return TransactionResultEventType.BAKER_STAKE_DECREASED;
+        return TransactionResultEventType.DELEGATION_STAKE_DECREASED;
```

Comment:
Please add this bugfix to the changelog. Something like: Fixed incorrect transaction event with respect to a transaction status for when a delegator decreased its stake.